modules/tinymce/src/core/main/ts/keyboard/NavigationUtils.ts

```diff
@@ -59,14 +81,20 @@ const moveHorizontally = (editor: Editor, direction: HDirection, range: Range, i
   }
 
   if (isBeforeFn(nextCaretPosition)) {
-    return FakeCaretUtils.showCaret(direction, editor, nextCaretPosition.getNode(!forwards) as HTMLElement, forwards, false);
+    return getAdjacentFloating(nextCaretPosition, direction).fold(
```

Comment:
Could we name this `getAdjacentAbsPositionedElement` or something since floating could mean that we are looking for `float: left` or something? Maybe that becomes to long might be a better name.